### ISSUE TITLE
Remove Color Contrast Failure PHP file

### ISSUE THREAD

**Issue by SteveJonesDev** [equalizedigital/accessibility-checker#618]
Remove file `color_contrast_failure.php` and ensure it doesn't cause any issues.
